src/backend/sqlite/table.rs

```diff
@@ -102,7 +102,7 @@ impl TableBuilder for SqliteQueryBuilder {
                     _ => "blob".into(),
                 },
                 ColumnType::VarBinary(length) => format!("binary({})", length),
-                ColumnType::Boolean => "integer".into(),
+                ColumnType::Boolean => "bool".into(),
```

Comment:
Hey @anshulxyz, sorry for the long delay! Thank you so much for the contributions!

I agree with this breaking changes. But I prefer mapping it as `boolean`. Just to be aligned with the type affinity listed on SQLite docs: https://www.sqlite.org/datatype3.html#affinity_name_examples

```suggestion
                ColumnType::Boolean => "boolean".into(),
```